common/flatpak-installation.c

```diff
@@ -1648,6 +1648,33 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_default_languages:
+ * @self: a #FlatpakInstallation
+ * @error: return location for a #GError
+ *
+ * Get the default languages used by the installation to decide which
+ * subpaths to install of locale extensions. This list may also be used
+ * by frontends like GNOME Software to decide which language-specific apps
+ * to display.
+ *
+ * Returns: (array zero-terminated=1) (element-type utf8) (transfer full):
+ *   A possibly empty array of locale strings, or %NULL on error.
+ * Since: 1.4
```

Comment:
Should be `Since: 1.4.3` I think. That will be the first release to have this API.